### DISCUSSION TITLE
How to get Live.Avalonia to support hot reload (in F#)

### DISCUSSION THREAD

**Discussion by houstonhaynes** [JordanMarr/ReactiveElmish.Avalonia#55] [Q&A]
I'm having a devil of a time getting Live.Avalonia to light up. 

https://github.com/AvaloniaUI/Live.Avalonia

the "RxApp" reference in their example seems to be begging for ReactiveUI but there's a conflict with Avalonia.ReactiveUI and I'm kind of stumped as to how to resolve it. Can you give it a looksee? 🙏

**Comment by houstonhaynes** [comment]
UPDATE I just found an F# example in the repo - will sniff around that 

https://github.com/AvaloniaUI/Live.Avalonia/blob/main/Live.Avalonia.FuncUI.Sample

**Comment by JordanMarr** [comment]
I’ve been too overloaded in work projects over the last couple months to so much in the evenings. 

I wonder if this could be fixed by simply updating to use the latest Avalonia versions.

**Reply by houstonhaynes** [reply]
No sweat at all. I've set it aside for now and am focusing on getting the Material.Avalonia sample app to line up
